### PR TITLE
Detect Framer

### PR DESCRIPTION
**Pull request by ankon**
This _should_ allow to detect Framer in the overview. 

I wasn't able to run the query though, but my BigQuery foo isn't strong enough to figure out what's going on.

cc @kurtextrem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,5 +18,5 @@ If you'd like to add a new host, you'll need to do three things:
 
 List of available datasets:
 
-* [chrome-ux-report](https://bigquery.cloud.google.com/dataset/chrome-ux-report:all?pli=1)
-* [httparchive](https://bigquery.cloud.google.com/dataset/httparchive:summary_requests?pli=1)
+* [chrome-ux-report](https://console.cloud.google.com/bigquery?p=chrome-ux-report&d=all&page=dataset)
```
Comment (tunetheweb) [inline README.md:21]: Let's use the materialized view:

```suggestion
* [chrome-ux-report](https://console.cloud.google.com/bigquery?p=chrome-ux-report&d=materialized&page=dataset)
```

**Comment by ankon** [comment]
> I wasn't able to run the query though, but my BigQuery foo isn't strong enough to figure out what's going on.

Needed to switch the date to 2024-03-01, thanks @kurtextrem.

File: ttfb.sql [hunk 2]
```diff
@@ -11,6 +11,8 @@ DECLARE PLATFORMS ARRAY<STRUCT<regex STRING, name STRING>> DEFAULT [
   ('wordpress.com', 'Automattic'),
   ('a9130478a60e5f9135f765b23f26593b', 'Automattic'),
   ('flywheel', 'Flywheel'),
+  ('caddy/', 'Framer'),
```
Comment (tunetheweb) [inline ttfb.sql:14]: Is "Caddy" to do with CaddyServer? Are those ALWAYS framer websites?

**Review by tunetheweb — CHANGES_REQUESTED** [review]

File: ttfb.sql (hunk 2 above)
Comment (ankon) [reply ttfb.sql:14]: Caddy indeed has something to do with Caddy :)

By default Caddy uses just `Server: Caddy`, we're returning `Caddy/opaque-versiony-thing` at the moment. In hindsight that might sound a bit like a bad choice, we used it though to recognize the platform while we're switching. 

Plan is to go back to just `Framer/` as prefix, as soon as we've done that I can file an update to drop this one again.

Comment (tunetheweb) [reply ttfb.sql:14]: OK I ran the below query to compare this header with Wappalyzer detection (that doesn't use this header) and seems like the vast majority of these are Framer sites (and the others might well be false negatives). So LGTM.

image
 

```SQL
SELECT
  client,
  COUNTIF(p.page IS NOT NULL) as framer_pages,
  COUNTIF(p.page IS NULL) as not_framer_pages
FROM
  `httparchive.all.requests` r
LEFT OUTER JOIN
  (
  SELECT date, client, page, root_page
  FROM
    `httparchive.all.pages` p,
  UNNEST (technologies) as tech
  WHERE date = "2024-03-01" AND
  tech.technology = 'Framer Sites'
  ) p
USING (date, client, page, root_page)
WHERE
  date = "2024-03-01" AND
  is_main_document AND
  LOWER(CONCAT(IFNULL(JSON_VALUE(r.summary, '$.resp_x_powered_by'), ''), IFNULL(JSON_VALUE(r.summary, '$.resp_via'), ''), IFNULL(JSON_VALUE(r.summary, '$.resp_server'), ''))) LIKE 'caddy/%'
GROUP BY client
```

**Review by tunetheweb — APPROVED** [review]

**Review by rviscomi — APPROVED** [review]
Thanks both!

**Pull request merged**